crates/bevy_ecs/src/storage/table/column.rs

```diff
@@ -91,17 +95,23 @@ impl ThinColumn {
         &mut self,
         last_element_index: usize,
         row: TableRow,
-    ) {
-        let _ = self
+    ) -> OwningPtr {
+        let data = self
             .data
             .swap_remove_unchecked(row.as_usize(), last_element_index);
         self.added_ticks
-            .swap_remove_unchecked(row.as_usize(), last_element_index);
+            .swap_remove_unchecked(row.as_usize(), last_element_index)
+            .read();
```

Comment:
I think you can revert the changes to add `.read()` now that you aren't trying to return the change ticks.